tests/upgrades/test_capsule.py

```diff
@@ -265,11 +252,11 @@ def test_post_user_scenario_capsule_sync_3(self):
             content_view=content_view,
         ).create()
         content_view.publish()
-        promote(content_view.read().version[0], self.lc_env.id)
+        promote(content_view.read().version[0], lc_env.id)
 
         # Run a Capsule sync
-        capsule = entities.SmartProxy().search(query={'search': f'name={self.cap_host}'})[0]
-        call_entity_method_with_timeout(entities.Capsule(id=capsule.id).content_sync, timeout=1500)
+        capsule = entities.SmartProxy().search(query={'search': f'name={cap_host}'})[0]
+        call_entity_method_with_timeout(entities.Capsule(id=capsule.id).content_sync, timeout=3600)
         sync_status = entities.Capsule(id=capsule.id).content_get_sync()
-        self.assertEqual(len(sync_status['active_sync_tasks']), 0)
-        self.assertEqual(len(sync_status['last_failed_sync_tasks']), 0)
+        assert len(sync_status['active_sync_tasks']) == 0
+        assert len(sync_status['last_failed_sync_tasks']) == 0
```

Comment:
non-blocking, but looks better imo
```suggestion
        assert sync_status['active_sync_tasks']
        assert sync_status['last_failed_sync_tasks']
```

Reply:
fixed.